app/src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,9 @@
         <activity android:name=".view.activities.RequestDetailActivity" />
         <activity android:name=".view.activities.MemberProfileActivity" />
         <activity android:name=".view.activities.SendRequestActivity" />
-        <activity android:name=".view.activities.SettingsActivity"/>
+        <activity android:name=".view.activities.SettingsActivity" />
```

Comment:
Did you intend to change line 34 here as well? Please make sure that when you autoformat your code it only changes the lines that you have worked on.